Content.Client/Doors/DoorSystem.Firelock.cs

```diff
@@ -22,15 +19,19 @@ private void OnAppearanceChange(EntityUid uid, FirelockComponent comp, ref Appea
         var boltedVisible = false;
         var unlitVisible = false;
 
-        if (!_appearanceSystem.TryGetData<DoorState>(uid, DoorVisuals.State, out var state, args.Component))
+        if (!Appearance.TryGetData<DoorState>(uid, DoorVisuals.State, out var state, args.Component))
             state = DoorState.Closed;
 
-        boltedVisible = _appearanceSystem.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component) && lights;
+        boltedVisible =
+            Appearance.TryGetData<bool>(uid, DoorVisuals.BoltLights, out var lights, args.Component) && lights;
         unlitVisible =
             state == DoorState.Closing
-            ||  state == DoorState.Opening
-            ||  state == DoorState.Denying
-            || (_appearanceSystem.TryGetData<bool>(uid, DoorVisuals.ClosedLights, out var closedLights, args.Component) && closedLights);
+            || state == DoorState.Opening
+            || state == DoorState.Denying
+            || Appearance.TryGetData<bool>(uid,
+                DoorVisuals.ClosedLights,
+                out var closedLights,
+                args.Component) && closedLights;
```

Comment:
```suggestion
            state == DoorState.Closing ||
            state == DoorState.Opening ||
            state == DoorState.Denying ||
            Appearance.TryGetData<bool>(uid,
                DoorVisuals.ClosedLights,
                out var closedLights,
                args.Component) &&
            closedLights;
```
indentation check again, please.